blazor/rating/templates.md

```diff
@@ -132,9 +130,7 @@ Below example demonstrates the Full Template of Rating.
 
 ## Emoji icon
 
-You can also use Emoji icon as a rating item.
-
-Below example demonstrates the Emoji Icon of Rating.
+You can also use Emoji icon as a rating items by using the `EmptyTemplate` tag directive. Which can be any Emoji icons you want.
```

Comment:
You can use emojis of your choice as rating symbol by specifying them as template content within the EmptyTemplate tag directive.

Reply:
updated as per comments